packages/utils/schema/user-profile.graphql

```diff
@@ -7,7 +7,7 @@ type Profile
   avatar: ImageMetadata
   backgroundImage: ImageMetadata
   description: String @string(maxLength: 42000)
-  emoji: String @string(maxLength: 2)
+  emoji: String @string(maxLength: 3)
```

Comment:
OpenMoji names their SVGs of emojis based on the characters. There's a `F9D1-1F3FB-200D-2764-FE0F-200D-1F48B-200D-1F9D1-1F3FF.svg`, *(a light skinned man loves a dark skinned woman)*, meaning there's at least series of ten.

There's no indication the Unicode people are going to tire of coming up with these strings, so why not do something like 20?